src/__tests__/CurrencyPair.test.js

```diff
@@ -0,0 +1,26 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom';
+import '@testing-library/jest-dom/extend-expect';
+import CurrencyPair from '../components/CurrencyPair';
+
+describe('CurrencyPair component test', () => {
+  it('Render Currency pair', () => {
+    const exchangeData = {
+      ticker: 'USD/JPY',
+      bid: '134.060',
+      ask: '134.060',
+      open: '134.122',
+      low: '133.984',
+      high: '134.138',
+      changes: -0.06200000000001182,
+      date: '2023-04-18 18:48:20',
+    };
+    const { container } = render(
+      <BrowserRouter>
+        <CurrencyPair exchangeItem={exchangeData} />
+      </BrowserRouter>,
+    );
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
- [ ] Dear @zdnahom, I notice you have added only `Snapshot` tests to your project. Please remember it's mandatory to test out several components of the app and some pure functions in Redux code. Also please consider adding integration tests using React Testing Library.

Reply:
Thanks for your review @TadesseDev, I will add more tests👍

Reply:
You are doing great @zdnahom. 🙏